app/services/PsrRetrievalService.scala

```diff
@@ -92,7 +92,9 @@ class PsrRetrievalService @Inject()(
 
             transformedMemberDetailsUa <- psrDetails.membersPayments
               .map(
-                memberPayments => memberPaymentsTransformer.transformFromEtmp(transformedAssetsUa, srn, memberPayments)
+                memberPayments =>
+                  memberPaymentsTransformer
+                    .transformFromEtmp(transformedAssetsUa, request.previousUserAnswers, srn, memberPayments)
```

Comment:
I cannot see what triggers request.previousUserAnswers to get populated (i.e. via IdentifyAndRequireData), not sure if I'm missing something?

Reply:
Added